XGDB/jobs/apps_exec.php

```diff
@@ -71,9 +71,15 @@
     }
     $add_app="INSERT INTO $global_DB.apps (app_id, program, platform, nodes, proc_per_node, date_added, description, developer, is_default, max_job_time) values ('$app_id', '$program', '$platform', $nodes, $proc_per_node, now(), '$description', '$developer', '$is_default', '$max_job_time')"; // ADMIN hard-coded on purpose
     $insert=mysql_query($add_app);
-    if($insert)
+    
+    if($insert) # 2-17-16 added this loop which will always retrieve the latest uid (for highlighting) even if records were previously deleted.
     {
-        header("location:apps.php#${next_uid}");
+        $max_query="SELECT MAX(uid) as max_uid from ${global_DB}.apps";

```

Comment:
Understood. People argue all the time about whether opening braces should be at the end of the line or by themselves on the next line, and that's not something I'm too worried about. As consistent as we can be, great, but with a codebase as big and complex as this some inconsistencies can be expected.

But indentation should always reflect logical structure, and before this last commit the indentation made you look for structure that wasn't there. Takes a lot of effort for the reader to sort it out. It's much easier to read now!